src/pages/how-to/netflow.mdx

```diff
@@ -0,0 +1,248 @@
+# Introduction
+
+NetBird’s **Traffic Events** feature provides a high-level view of traffic flows between network peers and resources.
+It captures connection events on the client (peer) side – for example, when `Peer A` connects to `Peer B` –
+allowing administrators to observe how devices communicate across the NetBird network​.
+The primary purpose of flow logging is to help network admins monitor network activity,
+detect unusual or unauthorized connections, and troubleshoot connectivity issues in their NetBird mesh VPN​.
+Unlike packet capture, **Traffic Events** record metadata about the traffic (addresses, ports, timestamps, etc.) rather than the contents,
+preserving privacy while still giving useful insight​.
```

Comment:
It is not really about the mesh network. The user will have an impression that we support it on for p2p. We need to emphasise both cases: p2p and access to internal resources (maybe even crosslinking to Resources in the networks feature doc). I'd also emphasise more user to resource as this is the most common case for business